packages/affine/model/src/consts/connector.ts

```diff
@@ -11,6 +11,14 @@ export enum PointStyle {
   Triangle = 'Triangle',
 }
 
+export const PointStyleMap = Object.entries(PointStyle).reduce(
+  (acc, [key, value]) => {
+    acc[value] = key as keyof typeof PointStyle;
+    return acc;
+  },
+  {} as Record<PointStyle, keyof typeof PointStyle>
+);
+
```

Comment:
A general function can be added

Reply:
Thanks for your comment! I have add a `createEnumMap` in utils.